news/2025/2025-07-18-osucatch-world-cup-2025-finals-recap.md

```diff
@@ -0,0 +1,114 @@
+
+Coming into this match, it was very difficult to give a clear edge to either team. Throughout the tournament, Chile had focused on the Double Time, Mixed Mod and Hidden brackets, while Finland opted for the more gimmicky picks, as well as the Mixed Mod and control Double Time picks. Chile were favoured on maps such as [DT1](https://osu.ppy.sh/beatmapsets/2398898#fruits/5199042), which was promptly banned by Finland, who themselves saw their strongest weapon, [NM3](https://osu.ppy.sh/beatmapsets/2398883#fruits/5199003), banned by Chile. Most maps left would be very contested either way, and the teams knew it.
+
+Chile looked comfortable at the start of the match, with back-to-back picks won by more than 100k points, in [HD1](https://osu.ppy.sh/beatmapsets/2398887#fruits/5199012) and [DT3](https://osu.ppy.sh/beatmapsets/2398909#fruits/5199074). On the other side, Finland struggled to find their footing — their [NM4](https://osu.ppy.sh/beatmapsets/2398973#fruits/5199190) pick won by barely more than 2k points, but they went on to lose [MM2](https://osu.ppy.sh/beatmapsets/2398941#fruits/5199115), conceding an early break point.
+
+They were quick to get back on their feet though, taking Chile's [MM3](https://osu.ppy.sh/beatmapsets/2399039#fruits/5199376), followed by [DT4](https://osu.ppy.sh/beatmapsets/2399031#fruits/5199360) to bring the scoreline back to even. The next couple of picks would be quite chaotic — Chile would once again concede a break point, this time on [HD2](https://osu.ppy.sh/beatmapsets/2398903#fruits/5199063), by just 611 points. But Finland failed to capitalise on it by also losing their own [NM2](https://osu.ppy.sh/beatmapsets/2398885#fruits/5199008) pick.
+
+Seeing as things could swing either way depending on their performance at the moment, picking proved to be a difficult task for both of the teams. But Chile's [MM1](https://osu.ppy.sh/beatmapsets/2398907#fruits/5199071) and Finland's [NM1](https://osu.ppy.sh/beatmapsets/2398958#fruits/5199148) would go to their respective teams, leaving the score at 5–5. However, Chile's next [HD1](https://osu.ppy.sh/beatmapsets/2398911#fruits/5199078) pick would be their biggest mistake this tournament, letting Finland get a break point, allowing them to close out the match 7–5 on [HR3](https://osu.ppy.sh/beatmapsets/2398918#fruits/5199091) by only 575 points.
```

Comment:
starting a sentence with "but" is technically not correct form, so doing it excessively doesn't feel right

```suggestion
Seeing as things could swing either way depending on their performance at the moment, picking proved to be a difficult task for both of the teams. Nevertheless, Chile's [MM1](https://osu.ppy.sh/beatmapsets/2398907#fruits/5199071) and Finland's [NM1](https://osu.ppy.sh/beatmapsets/2398958#fruits/5199148) would go to their respective teams, leaving the score at 5–5. However, Chile's next [HD1](https://osu.ppy.sh/beatmapsets/2398911#fruits/5199078) pick would be their biggest mistake this tournament, letting Finland get a break point, allowing them to close out the match 7–5 on [HR3](https://osu.ppy.sh/beatmapsets/2398918#fruits/5199091) by only 575 points.
```